test-site/scripts/build.sh

```diff
@@ -7,4 +7,6 @@ set_working_dir_to_test_site () {
 
 set_working_dir_to_test_site
 
-npx jambo build && npx grunt webpack
+npx jambo build
+chmod u+x ./public/static/node_modules/esbuild/bin/esbuild
```

Comment:
since this might just be a local issue, I'm not sure we want to commit this change. do you know if this is an issue if you make a fresh clone of the repo?

Reply:
It wouldn't have any issue if it's a fresh clone. As long as I run `npm i` in static/, it'll error out.